app/commands/user_command.rb

```diff
@@ -39,14 +39,16 @@ def efficiently_delete_relations(time = Time.now)
     Playlist.joins(:assets).where(assets: { user_id: user.id })
             .update_all(['tracks_count = tracks_count - 1, playlists.updated_at = ?', time])
     Track.joins(:asset).where(assets: { user_id: user.id }).update_all(deleted_at: time)
-    Comment.joins("INNER JOIN assets ON commentable_type = 'Asset' AND commentable_id = assets.id")
```

Comment:
This seems to be getting user's comments on assets specifically, while `[comments].each do |user_relation|` would get both asset and blog comments...

How certain are we that it's a necessary extra step?

Reply:
Comments originally were polymorphic in practice. These days, they aren't. I believe without cleaning the db, we might run into bugsnags, specifically on pages like `sudara/comments` unless we clean up the db/relationship.